swift/apple/.gitignore

```diff
@@ -5,5 +5,4 @@ DerivedData/
 xcuserdata/
 **/*.xcuserstate
 
-Firezone/xcconfig/config.xcconfig
```

Comment:
Ok so this change simplifies things nicely. Instead of defining multiple xcconfigs and moving them into place just before build, we simply specify the build vars on the command directly. So we can now commit a single xcconfig and use that for development and the release build scripts will override what vars they need.